### PR TITLE
Add yomi miners

### PR DESCRIPTION
**Pull request by gsanchietti**
The new miners will analyze the usage of yomi-rspamd plugin.

New mines:
- total analyzed mail
- total virus found
- total suspicious found

See: https://github.com/nethesis/yomi-rspamd

Nethesis/dev#5752

### CODE REVIEW

**Comment by gsanchietti** [comment]
@filippocarletti and @cotosso could you please check the widget labels?

**Review by edospadoni — APPROVED** [review]

File: beatrice/public/i18n/en/yomitotal-counter.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"title":"Total mail analyzed by Yomi sandbox"}
```
Comment (andre8244) [inline beatrice/public/i18n/en/yomitotal-counter.json]:
```suggestion
{"title":"Total mails analyzed by Yomi sandbox"}
```

File: ciacco/miners/yomitotal-counter [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+if [ -f "$PREVIOUS_LOG" ]; then
+    zgrep -E "$DAY" $PREVIOUS_LOG*  > $TMP_FILE
+fi
+
+# Match also dates like "Jul  1"
+grep -E "$DAY" /var/log/maillog >> $TMP_FILE
+
+value=0
+
+value=$(grep 'rspamd_task_write_log' /var/log/maillog | grep -e "YOMI_WAIT" -e "YOMI_CLEAN" -e "YOMI_SUSPICIOUS" -e "YOMI_VIRUS" $TMP_FILE | wc -l)
```
Comment (andre8244) [inline ciacco/miners/yomitotal-counter]:
```suggestion
value=$(grep 'rspamd_task_write_log' /var/log/maillog | grep -e "YOMI_WAIT" -e "YOMI_UNKNOWN" -e "YOMI_CLEAN" -e "YOMI_SUSPICIOUS" -e "YOMI_VIRUS" $TMP_FILE | wc -l)
```

**Review by nrauso — APPROVED** [review]
Good for me!

**Comment by gsanchietti** [comment]
Merge after release of blacklists

**Pull request merged**